_1327/static/less/1327.less

```diff
@@ -220,3 +220,11 @@ footer a {
 table.table-narrow {
 	max-width: 400px;
 }
+
+// Overwrite bootstrap table coloring
+.table-striped > tbody > tr:nth-of-type(odd) {
+	background-color: @white;
+}
```

Comment:
i still need the definition for even rows. i thought having both at the same place would be nice. what do you think?